jax/random.py

```diff
@@ -200,32 +207,51 @@ def _broadcast(x):
       _threefry2x32_gpu_translation_rule
 
 @jit
+@partial(vmap, in_axes=(None, 0))
```

Comment:
Instead of using `vmap` here, can we just rely on the (elementwise) implementation broadcasting along leading axes? That is, is the `vmap` necessary (even if we tweak the `np.stack` calls or something)? As written with `vmap`, it means the function is no longer rank-polymorphic IIUC, and has a bit more complexity.